core/store/src/trie/mod.rs

```diff
@@ -498,6 +498,16 @@ pub struct TrieRefcountChange {
     rc: std::num::NonZeroU32,
 }
 
+impl TrieRefcountChange {
+    pub fn hash(&self) -> CryptoHash {
+        self.trie_node_or_value_hash
+    }
+
+    pub fn value(&self) -> Vec<u8> {
+        self.trie_node_or_value.clone()
+    }
```

Comment:
Those should return references.  Caller can always clone but if caller only needs a reference providing methods which clone does unnecessary copying.  Also, perhaps `payload` would be better than `value`? Value suggest that it’s a value stored in the trie whereas it may be a node.


```suggestion
    pub fn hash(&self) -> &CryptoHash {
        &self.trie_node_or_value_hash
    }

    pub fn payload(&self) -> &[u8] {
        self.trie_node_or_value.as_slice()
    }
```